bin/pre-evacuation-setup-service

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+import logging
+import os
+import socket
+import subprocess
+
+SOCKET_PATH = os.path.join(os.environ["SNAP_DATA"], "data", "shutdown.sock")
+
+def stop_nova_compute():
```

Comment:
The hypervisor snap uses the snaphelpers library to manage snap related commands. Can you look into this library to manage the state of the nova-compute service?